WBooks/AppDelegate.swift

```diff
@@ -7,33 +7,74 @@
 //
 
 import UIKit
+import GoogleSignIn
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate {
-
+class AppDelegate: UIResponder, UIApplicationDelegate, GIDSignInDelegate {
+    
     var window: UIWindow?
-
+    
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        if Environment.isTestTarget {
-            return true
-        }
+        window = UIWindow()
         
-        /*
-         * Uncomment this and remove this comment once Rollbar service is enabled
-         * and the keys are properly configured in the configuration files.
-         *
-        RollbarService().initialize()
-         *
-         */
-        
-        window = UIWindow(frame: UIScreen.main.bounds)
-        window?.rootViewController = ViewController()
-        window?.makeKeyAndVisible()
+        //Creates the navifationController and sets the first view.
+        let myViewController = LoginViewController()
+        window?.rootViewController = myViewController
         
+        // Initialize sign-in
+        GIDSignIn.sharedInstance().clientID = "770844318431-lbb1uq0ppnhpe4mvfd0ve03iqknp4sj5.apps.googleusercontent.com"
+        GIDSignIn.sharedInstance().delegate = self
```

Comment:
All initializations should be done before showing the first view of the app. The first view should be the last thing the app delegate do in this function.